SupportBank/Program.cs

```diff
@@ -1,12 +1,63 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Transactions;
 
 namespace SupportBank
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            
         }
     }
+
+    class Transaction
+    {
+        public Transaction(string date, string from, string to, string narrative, double amount)
+        {
+            Date = date;
+            From = from;
+            To = to;
+            Narrative = narrative;
+            Amount = amount;
+        }
+
+        public string Date { get;  }
+        public string From { get; }
+        public string To { get; }
+        public string Narrative { get; }
+        public double Amount { get; }
```

Comment:
It's best practice to declare properties above methods. The general order is:
- public properties
- private properties/variables
- public methods (constructor first if present and public)
- private methods

🙂